xscen/aggregate.py

```diff
@@ -457,3 +465,84 @@ def spatial_mean(
         ds_agg.attrs["cat:processing_level"] = to_level
 
     return ds_agg
+
+
+@parse_config
+def unstack_dates(
```

Comment:
Why is this in `aggregate`? Shouldn't it be in `utils`?

Reply:
Ouain, it could move. It's there because it's the kind of thing that is done that the same moment as aggregations.